Packs/Gurucul/ReleaseNotes/1_1_4.md

```diff
@@ -0,0 +1,75 @@
+#### Commands
+- Added gra-analytical-features-entity-value
+
+#### Incident Fields
+- **Breaking Change**:
+  - Added GRA Case Risk Score
+  - Added GRA Case Weblink
+  - Updating GRA Case Open Date
+  - Updating GRA Case Risk Date  
+  - Updating GRA Case Status
+  - Updating GRA Case Entity Type Id
+  - Updating GRA Case Owner Id
+  - Updating GRA Case Anomaly Details
+  - Updating GRA Case Entity Id
+  - Updating GRA Case Entity
+  - Updating GRA Case Owner Name
+  - Updating GRA Case Owner Type
+  - Updating GRA Case Id
+  - Removed GRA Anomaly Status
+  - Removed GRA Anomaly Assignee
+  - Removed GRA Anomaly Name
+  - Removed GRA Anomaly Resource Name
+  - Removed GRA Anomaly Risk Score
+  - Removed GRA Anomaly Risk Accepted Date  
+  - Removed GRA Anomaly Assignee Type
+
+#### Classifiers  
+##### The renamed classifier 
+  - classifier-GRACase.json
+  - classifier-GRACase_5_9_9.json  
+  - GRACase-Classifier  
+
+##### The removed classifier
+  - classifier-GRAHighRiskUser.json
+  - classifier-GRAHighRiskUser_5_9_9.json
+  - GRAHighRiskUser-Classifier
+ 
+#### Mappers
+##### The renamed incoming mapper
+  - classifier-mapper-GRACases.json
+##### The removed incoming mapper
+  - classifier-mapper-GRAHighRiskUsers.json
+
+#### Incident Fields
+#### incident fields are renamed
+  - GRACase-Mapper
+#### incident fields are removed
+  - GRAHighRiskUser-Mapper
+
+#### Incident Types
+#### All incident types are renamed
+
+#### Layouts
+##### New: GRACaseLayout
+- %%UPDATE_RN%% (Available from Cortex XSOAR 6.0.0).
```

Comment:
Please update the release notes for the layout

Reply:
Added description and removed line